extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -117,6 +125,51 @@ export class Bind {
      */
     this.digestQueuedAfterScan_ = false;
 
+    /**
+     * @const @private {!Array<Node>}
+     */
+    this.dynamicRoots_ = [];
+
+    /**
+     * @const @private {!Array<Promise>}
+     */
+    this.mutationPromises_ = [];
+
+    this.subtreeMutationObserver_ = new MutationObserver(mutations => {
```

Comment:
According to http://caniuse.com/#feat=mutationobserver it's not supported pre Android 4.4 nor on Opera mini. I'm going to need to do some research on a fallback. Got any ideas?